README.md

```diff
@@ -1 +1,251 @@
-# empty-
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
```

Comment:
- [ ] Kindly consider explaining why the suggested changes are required as suggested by the previous reviewer.
you can write `Please remove all the commented-out code from the readme file before submitting them to your repository. 😄 because commented code make your codebase populated and it is against clean code`

Reply:
Thanks for letting me know, I already edited the message to the student. 👍